qt/scientific_interfaces/Indirect/ISISCalibration.cpp

```diff
@@ -487,13 +505,15 @@ void ISISCalibration::calSetDefaultResolution(const MatrixWorkspace_const_sptr &
 
       const auto energyRange = getXRangeFromWorkspace(ws);
       // Set default rebinning bounds
-      QPair<double, double> peakERange(-res * 10, res * 10);
+      QPair<double, double> peakERange(-res * 10 + (energyRange.second + energyRange.first) / 2.0,
+                                       res * 10 + (energyRange.second + energyRange.first) / 2.0);
       auto resPeak = m_uiForm.ppResolution->getRangeSelector("ResPeak");
       setPlotPropertyRange(resPeak, m_properties["ResELow"], m_properties["ResEHigh"], energyRange);
       setRangeSelector(resPeak, m_properties["ResELow"], m_properties["ResEHigh"], peakERange);
 
       // Set default background bounds
-      QPair<double, double> backgroundERange(-res * 9, -res * 8);
+      QPair<double, double> backgroundERange(-res * 20 + (energyRange.second + energyRange.first) / 2.0,
+                                             -res * 15 + (energyRange.second + energyRange.first) / 2.0);
```

Comment:
The term (energyRange.second + energyRange.first) / 2.0 is calculated four times within a few lines of code. It could be calculated once and then re-used.